src/components/ErrorCard.tsx

```diff
@@ -0,0 +1,26 @@
+import { Dispatch, SetStateAction } from "react";
+
+type ErrorCard = {
+  error: Error | null;
+  setError: any
```

Comment:
Set error must be of function type.

```suggestion
  setError: () => void;
```